src/lib.rs

```diff
@@ -3285,6 +3290,66 @@ impl Build {
         Ok(ret)
     }
 
+    fn apple_deployment_version(
+        &self,
+        os: AppleOs,
+        target: &str,
+        arch_str: Option<&str>,
+    ) -> String {
+        fn rustc_provided_target(rustc: Option<&str>, target: &str) -> Option<String> {
+            let rustc = rustc?;
+            let output = Command::new(rustc)
+                .stderr(Stdio::piped())
```

Comment:
stdout/stderr are automatically set to piped when you use `.output()` so there's no need to specify them.